scripts/distribution/macOS-package/NodeConfigurationInstallerPlugin/NodeConfigurationInstallerPlugin/MyInstallerPane.m

```diff
@@ -152,24 +152,40 @@ - (BOOL) nodeNamesAreValid
     // Default maximum node name length that the collector-backend will accept. Measured in UTF-8 encoded bytes.
     int maxLen = 100;
 
+    // NB: Get the actual length in bytes if allowedChars is changed to contain non-ascii characters.
+    NSUInteger nodeNameLen = [nodeName length];
+
+    if (nodeNameLen == 0 && !(runOnStartup || runAfterInstall || reportToNetworkDashboard)) {
```

Comment:
I think it would be better if we disabled the entry field (so it is greyed out) when you disable all three options.
We should probably allow the field to be non-empty, so you can 
- write a node name `foo`
- disable all three options (see that `foo` is still there, but the field is _disabled_)
- enable at least one of the options again
- see that `foo` is set as the node name

Later in the process, we should then handle that the node is disabled. I believe the correct solution is to omit the `CONCORDIUM_NODE_INSTALL_{netname}_NODE_NAME` flag in the config file.